whatis277/harvest
# Fix login tokens migration name

Renames `tokens` to `login_tokens`

Updates existing migration instead of creating a new one
We're still just testing locally, so it's easy to fix 

Testing instructions:
1. `dc down --volumes`
2. `dc run --rm bean tern migrate`
3. `dc exec postgres psql -U postgres`
4. `\c bean_test`
5. `insert into login_tokens (email, hashed_token) values ('email', 'hashed token 1') on conflict (email) do update set hashed_token = 'hashed token 2' returning *;`
6. Ensure it inserts successfully
7. Wait a couple of minutes
8. `insert into login_tokens (email, hashed_token) values ('email', 'hashed token 1') on conflict (email) do update set hashed_token = 'hashed token 2' returning *;`
9. Ensure it updates the created_at and updated_at
10. Exit psql
11. `dc run --rm bean tern migrate --destination -6`
12. Ensure it migrates down successfully
